ovgs.proto

```diff
@@ -1,175 +1,218 @@
 // Copyright (c) 2023 Arista Networks, Inc.
 // Use of this source code is governed by the BSD 2-Clause License
 // that can be found in the LICENSE file.
-
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
-
 package ovgs.v1;
```

Comment:
do we expect that there will be a new 'package' if there is a revision?
(don't protos permit backward portability without such versioning?)

Reply:
so this is kinda just an external practice to sem version protos - we don't use internally but i don't think it is "against a style"

Reply:
ok, it seems weird, to me is all. I'd expect that either your proto changes are backwards compatible or you should make them backward compatible. I suppose if you don't control the whole ecosystem ("I can make the correct changes across  the whole surface of usage") then maybe:
  "use proto at version <commit hash>"
  
or:
  "use package name XXX.v##"
  
  that second seems super sketchy to me :( but I'm also far less used to external proto usages :(